server/src/main/java/org/apache/druid/discovery/BrokerClient.java

```diff
@@ -116,7 +116,7 @@ private Request getNewRequestUrl(Request oldRequest)
     catch (MalformedURLException e) {
       // Not an IOException; this is our own fault.
       throw DruidException.defensive(
-          "Failed to build url with path[%] and query string [%s].",
+          "Failed to build url with path[%s] and query string [%s].",
```

Comment:
I slipped in this unrelated one-liner fix in this PR (while looking into a flaky test).